plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2010,13 +2022,23 @@ public void registerCoreMembers() {
         // @Group core
         //
         // @Description
-        // TODO: Document Command Details
+        // Script injection is an alternative way to run task scripts. Includes the ability
+        // to use and modify established definitions and context contained in the script.
+        // Unlike the run command, inject keeps all definitions and queue properties.
+        // Inject will execute all those commands in the task script before continuing to the next commands.
+        // Inject is also using the same queue, meaning definitions within the injection are carried along.
+        // Inject keeps the same properties from the queue.

```

Comment:
This is an odd way of explaining this.
Inject is essentially taking another script and shoving it right on into the current script.
Inject does not keep definitions or use the same values - inject copies other commands into the current queue, and defs and all else are there because they are part of the queue so of course they're there.